collect_app/src/main/java/org/odk/collect/android/preferences/AboutPreferencesFragment.java

```diff
@@ -69,32 +68,23 @@ public void onStart() {
     }
 
     @Override
+    public void onDestroy() {
+        getActivity().unbindService(websiteTabHelper.getServiceConnection());
```

Comment:
There is a chance that `getActivity()` returns `null`, we should null check here.